### PR TITLE
feat: added meta tag to prevent page indexing

### PR DESCRIPTION
**Pull request by rahulkeswani101**
### Summary

Added a new meta tag to prevent page indexing.

#### Related Issues / PR's

https://github.com/SigNoz/engineering-pod/issues/1430

#### Screenshots

NA

### CODE REVIEW

File: frontend/public/robots.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # https://www.robotstxt.org/robotstxt.html
-User-agent: *
-Disallow:
+User-agent: DatadogSynthetics
```
Comment (vikrantgupta25) [inline frontend/public/robots.txt:2]: this is intended ?

Comment (rahulkeswani101) [reply frontend/public/robots.txt:2]: Yes, We don't want to block all crawlers from crawling. So, I thought to disallow  DatadogSynthetics only.

Comment (vikrantgupta25) [reply frontend/public/robots.txt:2]: any specific reason to not stop all the crawlers ? and only DD crawler ?

Comment (vikrantgupta25) [reply frontend/public/robots.txt:2]: i am not much aware of the context here.. @YounixM for your eyes

**Review by SagarRajput-7 — APPROVED** [review]

**Review by YounixM — APPROVED** [review]

**Pull request merged**